qldbdriver/qldbdriver.go

```diff
@@ -69,24 +69,23 @@ func New(ledgerName string, qldbSession *qldbsession.QLDBSession, fns ...func(*D
 	semaphore := sync2.NewSemaphore(int(options.MaxConcurrentTransactions), 0)
 	sessionPool := make(chan *session, options.MaxConcurrentTransactions)
 	isClosed := false
+	retryPolicy := RetryPolicy{MaxRetryLimit: 4, Backoff: ExponentialBackoffStrategy{SleepBase: 10000000, SleepCap: 5000000000}}
```

Comment:
You can't just give the default.

This needs to be set as the default in the driver options, and then grabbed from the options struct if a user wants to modify it. Also, are we not using time durations for sleep base and sleep cap?

Reply:
SleepBase and SleepCap are now time.Duration and when assigned to a number type, the number is interpreted as nano seconds hence why they are set to 10000000 and 5000000000 respectively.